src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+that row. Since our body is stretched across the entire viewport, the *track*
+containing the content (yellow in the picture above) will at least fill all
+available space, but will grow (and make the document scroll) if that's
+necessary to fit the content inside.
+
+For the columns we want to take a more dynamic approach: We want both nav and
+content to grow (and shrink), but we want nav to never go below 200px width and
+have content grow *more* than the nav. In Flexbox we would use flex-grow and
+flex-shrink here, but in Grid it's a little different:
+
+    body {
+      display: grid;
+      grid-template-rows: 150px auto 100px;
+      grid-template-columns: minmax(200px, 3fr) 9fr;
+    }
+
+We are defining 2 columns. The first column is defined using the minmax()
+function, which takes 2 values: The minimum and the maximum size of that track
+(it's like min-width and max-width in one). The minimum width is, as we
+discussed before, 200px. The maximum width is 3fr. Fr is a grid-specific unit
+that allows you distribute available space to the grid elements
+([fr probably stands for "fraction unit", but might also mean "free unit" soon](https://twitter.com/DasSurma/status/815910270023430144)).
+Our values here mean that both columns will grow to fill the screen, but the
+content column will always be 3 times as wide as the nav column (provided the
+nav column stays wider than 200px).
+
+While the *placement* of our grid items is not correct yet, the *size* of the
+rows and columns behave correctly and yield the behavior we were aiming for:
+
+<video controls autoplay loop muted poster="[url to poster image]">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/css-grid/responsive_vp8.webm" type="video/webm; codecs=vp8">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/css-grid/responsive_x264.mp4" type="video/mp4; codecs=h264">
+</video>
+
+## Placing the items
+
+One of the most powerful features of Grid is to be able to place items without
+any regards to the DOM order (although, because screen readers navigate the DOM,
+we highly recommend to be mindful about reordering elements to be properly
+accessible). If no manual placement is done, the elements are placed in the Grid
+in DOM order, arranged left to right and top to bottom. Each element occupying
+one *cell*. The order in which the grid is filled can be changed using
+[`grid-auto-flow`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow).
+
+So, how do we place elements? Arguably the easiest way to place grid items is by
+defining which columns and rows they cover. Grid offers two syntaxes to do this:
+In the first syntax you define start and end point. In the second one you define
+a start point and a span:
+
+    header {
+      grid-column: 1 / 3;
+    }
+    nav {
+      grid-row: 2 / span 2;
+    }
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/manualplacement.png">
```

Comment:
Don't include widths here, if you want this floated left or right, use `class="attempt-left|right"`